src/main/resources/db/changesets/20200305_trackliveviews.yaml

```diff
@@ -0,0 +1,25 @@
+databaseChangeLog:
+  - changeSet:
+      id: trackliveviews
+      author: danmoran
+      changes:
+        - addColumn:
+            tableName: dataset_table
+            columns:
+              - column:
+                  name: raw_table_name
+                  type: text
+              - column:
+                  name: soft_delete_table_name
+                  type: text
+        - sql:
+            comment: Backfill
+            sql: >
+              UPDATE dataset_table SET raw_table_name = name,
+              soft_delete_table_name = 'datarepo_sd_' || name
+        - addNotNullConstraint:
+            tableName: dataset_table
+            columnName: raw_table_name
+        - addNotNullConstraint:
+            tableName: dataset_table
+            columnName: soft_delete_table_name
```

Comment:
Should I add a `rollback` statement too? The other changesets I spot-checked didn't seem to include one.

Reply:
The bonus of using a declarative approach like this is that it knows how to apply the rollbacks from this without explicitly stating them.